ipaserver/install/server/install.py

```diff
@@ -384,6 +384,8 @@ def install_check(installer):
     print("  * Create and configure an instance of Directory Server")
     print("  * Create and configure a Kerberos Key Distribution Center (KDC)")
     print("  * Configure Apache (httpd)")
+    if options.setup_kra:
```

Comment:
Please update the man pages with `--setup-kra` option for both `ipa-server-install` and `ipa-replica-install`.